### DISCUSSION TITLE
Fatal error: Uncaught PDOException: Unknown column '_createdAt' in 'field list'

### DISCUSSION THREAD

**Discussion by mzungudev** [appwrite/appwrite#3923] [Q&A] [answered]
### 👟 Reproduction steps

Perform a fresh install of appwrite with following command found in documentation. with defaults for everything, port, hostname etc

```
docker run -it --rm \
    --volume /var/run/docker.sock:/var/run/docker.sock \
    --volume "$(pwd)"/appwrite:/usr/src/code/appwrite:rw \
    --entrypoint="install" \
    appwrite/appwrite:1.0.1
```

### 👍 Expected behavior

Console should be display at localhost

### 👎 Actual Behavior

404 not found. Docker logs on " appwrite/appwrite:1.0.1   "docker-php-entrypoi…"  reveals:

```
Worker 17 started successfully
Worker 23 started successfully
[Setup] - Server database init started...
[Setup] - Creating metadata table: appwrite...
[Setup] - Creating collection: databases...

Fatal error: Uncaught PDOException: Unknown column '_createdAt' in 'field list' in @swoole-src/library/core/Database/PDOStatementProxy.php:64
Stack trace:
#0 /usr/src/code/vendor/utopia-php/database/src/Database/Adapter/MariaDB.php(518): Swoole\Database\PDOStatementProxy->__call('execute', Array)
#1 /usr/src/code/vendor/utopia-php/database/src/Database/Database.php(1337): Utopia\Database\Adapter\MariaDB->createDocument('_metadata', Object(Utopia\Database\Document))
#2 /usr/src/code/vendor/utopia-php/database/src/Database/Database.php(560): Utopia\Database\Database->createDocument(Object(Utopia\Database\Document), Object(Utopia\Database\Document))
#3 /usr/src/code/app/http.php(162): Utopia\Database\Database->createCollection('databases', Array, Array)
#4 {main}
  thrown in @swoole-src/library/core/Database/PDOStatementProxy.php on line 64
[2022-09-24 14:08:48 #1.4]      ERROR   php_swoole_server_rshutdown() (ERRNO 503): Fatal error: Uncaught PDOException: Unknown column '_createdAt' in 'field list' in @swoole-src/library/core/Database/PDOStatementProxy.php:64
Stack trace:
#0 /usr/src/code/vendor/utopia-php/database/src/Database/Adapter/MariaDB.php(518): Swoole\Database\PDOStatementProxy->__call('execute', Array)
#1 /usr/src/code/vendor/utopia-php/database/src/Database/Database.php(1337): Utopia\Database\Adapter\MariaDB->createDocument('_metadata', Object(Utopia\Database\Document))
#2 /usr/src/code/vendor/utopia-php/database/src/Database/Database.php(560): Utopia\Database\Database->createDocument(Object(Utopia\Database\Document), Object(Utopia\Database\Document))
#3 /usr/src/code/app/http.php(162): Utopia\Database\Database->createCollection('databases', Array, Array)
#4 {main}
  thrown in @swoole-src/library/core/Database/PDOStatementProxy.php on line 64
```
### 🎲 Appwrite version

Version 1.0.x

### 💻 Operating system

MacOS

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by mzungudev** [comment]
Found relatively similar errors, but all had solutions of upgrading mariadb to 10.7, which is already the version that gets picked up by fresh install. 

I have also removed all containers and images from docker to have a clean slate and the error is still reproduced.

**Comment by stnguyen90** [comment, marked as answer]
@miguelangel-reyes, this error typically happens if there is a previous instance of Appwrite with an older version. If you're okay with clearing any previous data, run `docker compose down -v` to remove all containers and volumes. Then you can run `docker compose up -d` to start the stack again.

**Reply by mzungudev** [reply]
Thank you! I had removed all the appwrite containers and images (dockker RMI, etc) so figured that would give me a clean slate when I ran the install command. You suggestion got me running again. 

However, it may be a separate issue, but it always failed to log me in after I sing up for root account. Not sure if still something with a user db is messed up... I do see this in chrome developer console after trying to log in.

`Failed to load resource: the server responded with a status of 401 (Unauthorized)
app-all.js?v=500.1.0.1:4048 AppwriteException: User (role: guests) missing scope (account)
    at Proxy.<anonymous> (http://localhost/dist/scripts/app-all.js?v=500.1.0.1:38:32)
    at Generator.next (<anonymous>)
    at fulfilled (http://localhost/dist/scripts/app-all.js?v=500.1.0.1:3:98)`
